Mathlib/Topology/Perfect.lean

```diff
@@ -88,6 +87,24 @@ theorem preperfect_iff_nhds : Preperfect C ↔ ∀ x ∈ C, ∀ U ∈ 𝓝 x, 
   simp only [Preperfect, accPt_iff_nhds]
 #align preperfect_iff_nhds preperfect_iff_nhds
 
+section PerfectSpace
+
+variable (α)
+
+/--
+A topological space `X` is said to be perfect if its universe is a perfect set.
+Equivalently, this means that `𝓝[≠] x ≠ ⊥` for every point `x : X`.
+-/
+class PerfectSpace: Prop :=
+  univ_perfect' : Perfect (Set.univ : Set α)
```

Comment:
What about using `Preperfect` here, since the closedness condition is redundant?